spec/features/admin/merchants/index_spec.rb

```diff
@@ -21,4 +21,16 @@
     expect(page).to have_content(@merchant_6.name)
     expect(page).to_not have_content("Johnson-Dickinson")
   end
+
+  it "links to the merchant show page when you click the merchant's name" do
+    visit "/admin/merchants"
+
+    click_link(@merchant_1.name)
+    expect(current_path).to eq("/admin/merchants/#{@merchant_1.id}")
+
+    visit "/admin/merchants"
```

Comment:
This looks great - just a reminder that we can now use the pathing option with urls now. So this could also look like `visit admin_merchants_path`

Reply:
thank you!! I didn't think of using that within the scope of the testing suite